ci/build_wheel.sh

```diff
@@ -16,3 +16,6 @@ cd "${package_dir}"
 rapids-pip-retry wheel . -w "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}" -v --no-deps --disable-pip-version-check
 
 ../ci/validate_wheel.sh "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}"
+
+RAPIDS_PACKAGE_NAME="$(rapids-package-name wheel_python cuxfilter --pure --cuda "${RAPIDS_CUDA_VERSION}")"
+export RAPIDS_PACKAGE_NAME
```

Comment:
```suggestion
```

Was this left over from an earlier iteration?

It's at the end of this script that isn't called by anything else, so I don't think it'll actually affect anything.

Reply:
Oh I see, it's setting it for the benefit of this:

https://github.com/rapidsai/shared-workflows/blob/32156d9710c4a6994bb8bab8ddf1441aae03c12c/.github/workflows/wheels-build.yaml#L277

To be used as input in the upload step:

https://github.com/rapidsai/shared-workflows/blob/32156d9710c4a6994bb8bab8ddf1441aae03c12c/.github/workflows/wheels-build.yaml#L291

It wasn't obvious to me that this was modifying something in the environment for the next `shared-workflows` step to pick up. But I understand how that's working now.